tests/libnet/cloudinit/cloudinit.go

```diff
@@ -199,3 +199,9 @@ func CreateDefaultCloudInitNetworkData() string {
 	}
 	return data
 }
+
+func GetFedoraToolsGuestAgentBlacklistUserData(commands string) string {
+	return fmt.Sprintf(`#!/bin/bash
+            echo -e "\n\nBLACKLIST_RPC=%s" | sudo tee -a /etc/sysconfig/qemu-ga
+`, commands)
+}
```

Comment:
Can we use the builder pattern defined in this file to achieve the same?
Should we convert it to `WithSomething()`?

Reply:
no need imo, the usage is very specific and not common

Reply:
Cool, thank you for the feedback.
Otherwise everything looks good to me :+1: 

/approve